test/test_variant_assigner.py

```
@@ -5,122 +5,245 @@
-        expected_variants = [1, 0, 1, 1, 0, 0, 0, 2, 0, 2, 1, 0, 0, 1]
-
-        self.assert_variants(seeds, splits, expected_variants, var_assigner)
-
-    def assert_variants(self, seeds, splits, expected_variants, var_assigner):
-        for index, seed in enumerate(seeds):
-            frags = seed
-            split = splits[index]
-            variant = var_assigner.assign(split, frags[0], frags[1])
-            self.assertEqual(expected_variants[index], variant)
+def hash_unit(unit):
+    dig = hashlib.md5(str(unit).encode('utf-8')).digest()
```

Ruff (0.15.2)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)